cmds/tui.c

```diff
@@ -133,7 +133,7 @@ static const char *help[] = {
 	"ARROW         Navigation",
 	"PgUp/Dn",
 	"Home/End",
-	"Enter         Select/Fold",
+	"Enter         Graph fold/unfold or Select session",
```

Comment:
Wouldn't it be better saying "Fold/unfold graph" instead?

Reply:
I updated it to "Fold/unfold graph or Select session". Thanks.